LibreNMS/Alert/AlertUtil.php

```diff
@@ -36,14 +37,11 @@ class AlertUtil
     /**
      * Get the rule_id for a specific alert
      *
-     * @param  int  $alert_id
      * @return mixed|null
      */
-    private static function getRuleId($alert_id)
+    private static function getRuleId(int $alert_id)
```

Comment:
because the field is an integer, we know this will return an integer. (or null if not found)

0 and null are the same for the code, so setting the return type to int and casting to int, would be just fine here.